MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -454,6 +468,20 @@ - (void)displayLogFileFromDate:(NSDate *)logDate {
     [MMEEventLogger.sharedLogger readAndDisplayLogFileFromDate:logDate];
 }
 
+#pragma mark - MMEConfiguratorDelegate
+
+- (void)configurator:(id)updater didUpdate:(MMEEventsConfiguration *)configuration {
+    self.configuration = configuration;
+    [self.apiClient reconfigure:configuration];
+    [self.locationManager reconfigure:configuration];
```

Comment:
`reconfigure` is marked as `optional` - we should check to see if it responds to the selector (unless I'm looking at the wrong thing!)